### DISCUSSION TITLE
Increase the click area by an annotation point

### DISCUSSION THREAD

**Discussion by cyrilmarceau** [rnmapbox/maps#2772] [Ideas]
Hello,

Thanks for this library.

I use the annotation points because the markers don't seem to be anchored and when you move the map it floats on the map. So I use the point annotation but the click zone is very restricted to the point.

Is it possible to enlarge the click area via a prop like:

pressableAnchor={{top: myValue, left: myValue, bottom: myValue, right: myValue}}

Or something in that style.

Thanks,

**Comment by mfazekas** [comment]
@cyrilmarceau we don't have such a feature, you can try to add a transparent area around the point annotation, so it's bigger than actually, to increase hotbox.

**Comment by cyrilmarceau** [comment]
@mfazekas I see this issue on IOS but it' still present on Android: https://github.com/mapbox/mapbox-maps-ios/issues/979 and https://github.com/mapbox/mapbox-plugins-android/issues/1002

Is this corrected on new versions?

If not, would it be possible to fix this bug or reduce it as much as possible?
